bin/circleci/do-coverage

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+[ $CIRCLE_NODE_INDEX -eq 0 ] || exit 0
```

Comment:
Yes maybe in a future change we could do this; in which case it would probably not be lots of little files, but perhaps one big file with a switch statement.